test/integration/registerable_test.rb

```diff
@@ -1,6 +1,8 @@
 require 'test_helper'
 
 class RegistrationTest < Devise::IntegrationTest
+  uses_transaction :test_a_guest_user_should_receive_the_confirmation_instructions_from_the_default_mailer
```

Comment:
@ulissesalmeida It does not work very well, declare `uses_transaction` on one spec will break other specs, see travis build results.  I think AR just disables the transactional fixtures when you say `uses_transaction`...